zebra/zebra_evpn_mh.c

```diff
@@ -2101,8 +2103,7 @@ static void zebra_evpn_es_setup_evis(struct zebra_evpn_es *es)
 	}
 }
 
-static void zebra_evpn_flush_local_mac(struct zebra_mac *mac,
-				       struct interface *ifp)
+void zebra_evpn_flush_local_mac(struct zebra_mac *mac, struct interface *ifp)
```

Comment:
So both places where this function is called call zebra_evpn_mac_get_access_info where it is possible that a NULL pointer is returned for the ifp.  Can we put a guard in place to prevent that?  Is it actually possible for this to happen?